src/settings-ui/Settings.UI.Library/Telemetry/Events/OobeVariantAssignmentEvent.cs

```diff
@@ -0,0 +1,21 @@
+﻿// Copyright (c) Microsoft Corporation
+// The Microsoft Corporation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Diagnostics.Tracing;
+using Microsoft.PowerToys.Telemetry;
+using Microsoft.PowerToys.Telemetry.Events;
+
+namespace Microsoft.PowerToys.Settings.UI.Library.Telemetry.Events
+{
+    [EventData]
+    public class OobeVariantAssignmentEvent : EventBase, IEvent
+    {
+        public DateTime Date { get; set; }
+
+        public string AssignmentContext { get; set; }
```

Comment:
Still in discussion: Lilian initially stated that we needed ExperimentID, but there's no easy way to get ExperimentID from the code. The Azure ExP folk typically will use AssignmentContext and that should suffice.